app/src/main/res/layout-sw600dp-land/topic_info_fragment.xml

```diff
@@ -15,7 +15,7 @@
     android:orientation="horizontal">
 
     <ScrollView
-      android:layout_width="@dimen/elevation_0dp"
+      android:layout_width="@dimen/topic_info_fragment_scroll_view_width"
```

Comment:
This should actually be hardcoded `0dp` directly here. It should not be defined in `strings.xml` file.

Reply:
Okay will do the same for space_0dp